### PR TITLE
Replace curl with node-fetch

### PR DESCRIPTION
**Pull request by rtsao**
This PR changes the bot to kick off release verification builds with node-fetch rather than spawning a child process with curl.

### CODE REVIEW

**Review by lhorie — DISMISSED** [review]

File: src/release-verification.js [hunk 1]
```diff
@@ -64,22 +64,31 @@ module.exports = robot => {
...
+      output = await res.json();
+    } catch (err) {
+      // eslint-disable-next-line no-console
+      console.warn(err);
+    }
+
+    github.issues.createComment(
+      context.issue({
+        body: `Triggered Fusion.js build verification: ${output.web_url}`,
+      }),
```
Comment (rtsao) [inline src/release-verification.js:104]: Should this set the status message instead of commenting? @KevinGrandon

Comment (KevinGrandon) [reply src/release-verification.js:104]: Yeah, status URL is probably better here, happy to do that in a follow-up.

**Review by KevinGrandon — APPROVED** [review]
Thanks!

**Comment by KevinGrandon** [comment]
Thanks for taking this on, the tests are a little hairy, but happy to work with you on them later.

**Pull request merged**